packages/fx-core/templates/core/v3Migration/csharp.app.yml

```diff
@@ -0,0 +1,93 @@
+version: 1.0.0
+
+provision:
+{{#if activePlugins.fx-resource-bot}}
+  - uses: botAadApp/create # Creates a new AAD app for Bot Registration.
+    with:
+      name: {{appName}}bt$\{{RESOURCE_SUFFIX}}
```

Comment:
Nit: Just FYI, I saw multiple `\` used across the file content. Another way to make it clean is to use a different separator like this file: https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/templates/plugins/resource/cicd_v2/github/cd.yml

Reply:
thanks, I will have a try later